### PR TITLE
Autocmd: support multiple patterns

### PR DESCRIPTION
**Pull request by stsewd**
Patterns can be comma separated

### CODE REVIEW

File: grammar.js [hunk 1]
```diff
@@ -211,7 +211,7 @@ module.exports = grammar({
         optional(
           seq(
             $.au_event_list,
-            alias(/[a-zA-Z*.]+/, $.pattern),
+            commaSep1(alias(/[a-zA-Z*.]+/, $.pattern)),
```
Comment (stsewd) [inline grammar.js:214]: should this be grouped in a node maybe? `(patterns(pattern+))`

Comment (vigoux) [reply grammar.js:214]: Yup, better in a node.

Comment (vigoux) [reply grammar.js:214]: Not sure in the end...

Comment (vigoux) [reply grammar.js:214]: It's fine like this I think.

**Review by vigoux — APPROVED** [review]

**Pull request merged**